sonar-kotlin-plugin/src/test/java/org/sonarsource/kotlin/plugin/KotlinRulesDefinitionTest.kt

```diff
@@ -62,6 +62,15 @@ internal class KotlinRulesDefinitionTest {
         Assertions.assertThat(rule!!.securityStandards()).containsExactly("owaspTop10:a3")
     }
 
+    @Test
+    fun pciDssSecurityStandard() {
+        val repository: RulesDefinition.Repository? = repositoryForVersion(Version.create(9, 9))
+        val rule = repository?.rule("S6288")
+        Assertions.assertThat(rule).isNotNull
+        Assertions.assertThat(rule!!.securityStandards())
+            .containsExactly("cwe:522", "owaspTop10-2021:a4", "pciDss-3.2:6.5.8", "pciDss-4.0:6.2.4")
```

Comment:
I think you don't need to check for null with assertion. Let's just have :

```val rule = repository?.rule("S6288")!!```

and just drop the next line and non-null assertion on the line 71